iree/turbine/kernel/wave/scheduling/loop_reconstruction.py

```diff
@@ -99,39 +104,56 @@ def add_nodes_by_schedule(
             # Set the index for the new node by substituting the induction variable
             # for the current iteration.
             new_node.index = node.index
-            for dim in new_node.index:
-                new_node.index[dim] = new_node.index[dim].subs(
-                    {induction_variable: current_induction_variables[iteration]}
-                )
+            if new_node.index:
```

Comment:
are these for the new ops such as Register which doesnt have index?

Reply:
I hit this for the ExtractOps that don't have an index.